### PR TITLE
Move admin-subnet commands to support

### PR DESCRIPTION
**Pull request by anjalshireesh**
So that

`mc admin subnet register|health`

become

`mc support register|diagnostics`

The previous commands will print a message asking the user to use the
new ones.

### CODE REVIEW

File: cmd/admin-subnet-register.go [hunk 1]
```diff
@@ -18,160 +18,20 @@
...
-	Username string `json:"username"`
-	Password string `json:"password"`
-}
-
-// SubnetMFAReq - JSON payload of the SUBNET mfa api
-type SubnetMFAReq struct {
-	Username string `json:"username"`
-	OTP      string `json:"otp"`
-	Token    string `json:"token"`
+	Name:               "register",
```
Comment (vadmeste) [inline cmd/admin-subnet-register.go:26]: I think we have Hidden flag here, I guess you can use it, can you double check ?

File: cmd/admin-subnet-health.go [hunk 2]
```diff
@@ -18,653 +18,20 @@
...
-
-func greenText(s string) string {
-	console.SetColor("GREEN", color.New(color.FgGreen))
-	return console.Colorize("GREEN", s)
-}
-
-func warnText(s string) string {
-	console.SetColor("WARN", color.New(color.FgRed, color.Bold))
-	return console.Colorize("WARN", s)
+	Name:               "health",
```
Comment (vadmeste) [inline cmd/admin-subnet-health.go:26]: I think you can hide this omment with Hidden, can you check ?

**Review by vadmeste — COMMENTED** [review]
Few comments

File: cmd/admin-subnet-register.go (hunk 1 above)
Comment (anjalshireesh) [reply cmd/admin-subnet-register.go:26]: Yes, thanks. Done.

File: cmd/admin-subnet-health.go (hunk 2 above)
Comment (anjalshireesh) [reply cmd/admin-subnet-health.go:26]: Done

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**